ledger/participant-integration-api/src/main/scala/platform/apiserver/services/transaction/ApiTransactionService.scala

```diff
@@ -111,7 +111,9 @@ private[apiserver] final class ApiTransactionService private (
         TransactionFilter(request.parties.map(p => p -> Filters.noFilter).toMap),
         request.verbose,
       )
-      .via(logger.debugStream(transactionTreesLoggable))
+      .via(
+        logger.enrichedDebugStream("Responding with transaction tree.", transactionTreesLoggable)
```

Comment:
```suggestion
        logger.enrichedDebugStream("Responding with transaction trees.", transactionTreesLoggable)
```
For consistency, unless I'm missing something.